Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -51,5 +69,32 @@ public async Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync(DateTime? date
             var schedules = await _scheduleRepository.GetByDatePeriodAsync(userId, date.Value, daysNum);
             return _mapper.Map<ScheduleDTO>(schedules);
         }
+
+        private IEnumerable<ScheduleCandidateInfo> GetScheduleCandidateInfos(IEnumerable<Candidate> candidates, IEnumerable<AppointInterviewDTO> appointInterviewDTOs)
+        {
+            var scheduleCandidateInfos = new List<ScheduleCandidateInfo>();
+            foreach (var candidate in candidates)
+            {
+                var projectId = appointInterviewDTOs.Where(a => a.CandidateId == candidate.Id).Select(u => u.ProjectId).FirstOrDefault();
+                var projectResult = candidate.ProjectResults.Where(x => x.ProjectId == projectId).FirstOrDefault();
+                scheduleCandidateInfos.Add(new ScheduleCandidateInfo()
+                {
+                    BestTimeToConnect = candidate.BestTimeToConnect,
+                    Email = candidate.Email,
+                    Id = candidate.Id,
+                    Name = candidate.Name,
+                    ProjectResult = new ScheduleCandidateProjectResult()
```

Comment:
why not to use mapping?

Reply:
i have to change status and IsAssignedOnInterview in ProjectResult